lib/cb/config.rb

```diff
@@ -59,6 +59,7 @@ def set_default_api_uris
       @uri_application_create ||= '/cbapi/application/'
       @uri_application_external ||= '/v1/application/external'
       @uri_application_form ||= '/cbapi/job/:did/applicationform'
+      @uri_browser_id ||= '/consumer/browser-id'
```

Comment:
So I understand, this line has nothing to do with this PR, right? but it'll be part of your Feature Branch PR into master.